cli_tools/gce_ovf_import/domain/interfaces.go

```diff
@@ -53,5 +53,10 @@ type ImageImporterInterface interface {
 // If an import fails, all running imports will be cancelled, and images from finished imports
 // will be deleted.
 type MultiImageImporterInterface interface {
-	Import(ctx context.Context, params *OVFImportParams, fileURIs []string) (imageURIs []string, err error)
+	Import(ctx context.Context, params *OVFImportParams, fileURIs []string) (images []Image, err error)
+}
+
+// ImageDeleter checks whether images exist. If so, it deletes them.
+type ImageDeleter interface {
```

Comment:
Can this be a part of `cli_tools/common/domain`? There's nothing specific to OVF import in this interface and might be useful in other cases (although I don't see one right now).

Reply:
Moving sounds good to me; done!